Posit Infrastructure/How to find out the Session ID of your Posit Workbench session.md

```diff
@@ -0,0 +1,32 @@
+# Find out the Session ID of your Posit Workbench Session
+
+## Background
+
+When a Workbench Session is requsted by the user, it is assigned a Session ID.  This is a long string of letters and numbers that uniquely identify the session.  The Session ID corresponds to metrics and logs that are produced by the Posit Workbench application and collected for onward analysis and debugging of issues.
+
+## Steps to find out the Session ID of your Posit Workbench Session
+
+1. If you have not already done so, log into Posit Workbench and start a session, following the instructions at [https://github.com/Public-Health-Scotland/technical-docs/blob/main/Posit%20Infrastructure/How%20to%20Login%20to%20Posit%20Workbench.md] if necessary.
```

Comment:
```suggestion
1. If you have not already done so, log into Posit Workbench and start a session, following the instructions at [Technical-docs/How to login to Posit Workbench](https://github.com/Public-Health-Scotland/technical-docs/blob/main/Posit%20Infrastructure/How%20to%20Login%20to%20Posit%20Workbench.md) if necessary.
```

I was just going to suggest not having a bare link but I then realised the link doesn't work / the doc doesn't exist.